docs/data-access/data-download.md

```diff
@@ -0,0 +1,89 @@
+---
+id: data-download
+title: How to Download Data
+---
+
+The ARGO Data Platform uses the score-cient as a file download manager. Score facilitates the transfer of data with resumable download and built in BAM/CRAM slicing to make data download transfer fast and smooth.
+
+Please note:
+
+- Downloads are done in part and can be paused/resumed as needed
+- The score-client will automatically resume downloads if interrupted or paused briefly
+
+## Searching for Files
+
+Use the Platform [File Repository](https://platform.icgc-argo.org/repository) to search for a file set of interest. Query results can be narrowed down by selecting specific file values in the facet filters on the left side of the page. Once you have a file set identified, click `Download > File Manifest` on the top right of the table to download a TSV file manifest.
+
+The file manifest will contain a listing of the files that matched your query, along with some additional metatdata to assist in file identification. This file manifest will be used by the score-client as a list of files to download.
+
+> NOTE: Clinical data can be downloaded by any user. In order to download controlled molecular data, you **must have ICGC DACO approval**. Learn more about the [DACO application process here](data-access), or [apply for DACO approval here](https://icgc.org/daco).
+
+## Installing the Score-Client
+
+The score-client can be run in different ways depending on your operating system or setup:
+
+- If you are on Windows, use the Docker distribution
+- If you are on a Unix system (IOS/Linux) you can use the Docker distribution, or score-client directly.
+
+## Installing the Score-Client to Download Files
+
+### Docker Distribution
+
+Pull the latest version of the Score Docker distribution:
+
+```
+docker pull overture/score
+```
+
+Use the docker run command with the correct variables specified. You will need to define:
+
+- your personal [API Key](user-profile-and-token)
```

Comment:
instead of "your personal" should we say "your ARGO user API Key"?

Reply:
i dont think so - we want to emphasize that this is `only for you`, plus we have the link to the other page?

Reply:
could say your personal ARGO API Key.

Reply:
itsnt it redundant to say argo here?  i would never try and use another api token from any other site....

Reply:
ok yeah...I've never used one haha. Just wasn't sure if we needed to be specific because they have to find things in multiple places for this